ui/v2/src/components/Settings/SettingsAboutPanel.tsx

```diff
@@ -26,32 +28,72 @@ export const SettingsAboutPanel: FunctionComponent<IProps> = (props: IProps) =>
     );
   }
 
-  function renderVersion() {
-    if (!data || !data.version) { return; }
+  function maybeRenderLatestVersion() {
+    if (!dt || !dt.latestversion || !dt.latestversion.shorthash || !dt.latestversion.url) { return; }
+    if (!data || !data.version || !data.version.hash) {
+      return (
+        <>{dt.latestversion.shorthash}</>
+      );
+    }
+
+    if (data.version.hash !== dt.latestversion.shorthash) {
+      return (
+        <>
+          <strong>{dt.latestversion.shorthash} [NEW] </strong><a href={dt.latestversion.url}>Download</a>
+        </>
+      );
+    }
+
+    return (
+      <>{dt.latestversion.shorthash}</>
+    );
+  }
+
+  function RenderLatestVersion() {
     return (
-      <>
       <HTMLTable>
         <tbody>
-          {maybeRenderTag()}
           <tr>
-            <td>Build hash:</td>
-            <td>{data.version.hash}</td>
+            < td>Latest Version Build Hash: </td>
+            < td>{maybeRenderLatestVersion()} </td>
           </tr>
           <tr>
-            <td>Build time:</td>
-            <td>{data.version.build_time}</td>
+            <td><Button minimal={true} onClick={() => refetch()} text="Check for new version!" /></td>
```

Comment:
Having `minimal={true}` makes the button look weird in this context when not moused over. I'd get rid of it. I'm not sure that the exclamation mark is necessary, but that's just a subjective aesthetic opinion.

Reply:
i tried the combo minimal and ! and plain button without the !
i'll change back to plain button